src/main/java/sirius/biz/model/QueryController.java

```diff
@@ -209,7 +212,14 @@ public void queryApi(WebContext webContext, JSONStructuredOutput output) {
             output.beginObject("entity");
             output.property("id", entity.getId());
             for (Property property : determineVisibleProperties(descriptor)) {
-                output.property(property.getName(), property.getValue(entity));
+                if (property.is(AmountProperty.class)) {
+                    output.amountProperty(property.getName(),
+                                          Value.of(property.getValue(entity)).getAmount(),
+                                          NumberFormat.MACHINE_TWO_DECIMAL_PLACES,
```

Comment:
Is 2 decimal places enough for most usecases?

Reply:
maybe... now it´s the same behaviour as before...